pyodata/v2/model.py

```diff
@@ -426,7 +426,7 @@ def from_literal(self, value):
                 except ValueError:
                     raise PyODataModelError('Cannot decode datetime from value {}.'.format(value))
 
-        return value
+        return value.replace(tzinfo=datetime.timezone.utc)
```

Comment:
I would recommend for future readability to define this as constant with docstring pointing to odata v2 spec, explaining why the conversion happens instead of just call in return statement. 

Rationale:
- datetimes and timezones are bug creators and pains anyway
- In odata v4 this is different and the DateTime is pruned for this exact timezone reason.

Reply:
What conversion are you talking about? Default DateTime has tzinfo hodling None.
What OData V2 spec are you talking about? There are no mentions of Time zones in OData V2.

Reply:
The current version uses the function current_timezone(). If you want, you can think up a user configuration API which wouldn't be just a set of random functions.

Reply:
> What OData V2 spec are you talking about? There are no mentions of Time zones in OData V2.

Yes, it is quite hard to find true specification in the DateTime type in odata version 2.
[Odata.org](https://www.odata.org/documentation/odata-version-2-0/overview/)) pointed me towards 
[ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) or [W3C XML schema](https://www.w3.org/TR/xmlschema-2/#dateTime) and its lexical representations, which can specify timezone information in the suffix e.g. '2002-10-10T00:00:00+05:00'.

But seems that odata v2 expects just UTC (and even without daylight savings info, so one more reason for odata v4)

- Edm.DateTime '2000-12-12T12:00'. 
- Edm.DateTimeOffset '2002-10-10T17:00:00Z'.

But doesn't that means that similar conversion should not be only for DateTime but for DateOffset as well?

https://github.com/SAP/python-pyodata/blob/ce0b6cf6e43dbfeccdc5ea0b917c6780641fb160/pyodata/v2/model.py#L211

Reply:
Yep, DateTimeOffset expects UTC. I didn't touch DateTimeOffset, did I?